packages/terra-i18n/scripts/aggregate-translations/write-aggregated-translations.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+const fs = require('fs');
+const supportedLocales = require('../../src/i18nSupportedLocales');
+const generateTranslationFile = require('./generate-translation-file');
+
+const writeAggregatedTranslations = (aggregatedMessages, locales, fileSystem, outputDir) => {
+  // Make the output directory
+  if (fileSystem === fs) {
+    const outputDirectories = outputDir.split(path.sep);
+    let makeDir = outputDirectories.shift();
+
+    outputDirectories.forEach((dir) => {
+      makeDir = makeDir + path.sep + dir;
+      if (!fileSystem.existsSync(makeDir)) {
+        fileSystem.mkdirSync(makeDir);
+      }
+    });
+  } else {
+    fileSystem.mkdirpSync(outputDir);
+  }
+
+  // Create an file of aggregated translaiton messages for each locale
```

Comment:
This should be 'Create a file of aggregated translation messages for each locale'.